pyproject.toml

```diff
@@ -8,6 +8,7 @@ version = '0'
 
 [project.scripts]
 cockpit-bridge = "cockpit.bridge:main"
+cockpit-askpass = "cockpit.askpass:main"
```

Comment:
Can we tell this somehow to install into libexecdir? This doesn't belong into `$PATH`. I suppose not, but this seems easy enough to just do ourselves. This is just a single-line shell wrapper (`python3 -m cockpit.askpass`), or perhaps it can even be done with `#!/usr/bin/python3 ..`?

Perhaps the easiest way is to install it into libexecdir via the Debian/rpm packaging -- they know the destination path already, and an upstream `pip install` isn't something that any user would do, so it doesn't matter much where it goes there.